org.freedesktop.Tuhi.json

```diff
@@ -0,0 +1,69 @@
+{
+    "app-id": "org.freedesktop.Tuhi",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.32",
+    "sdk": "org.gnome.Sdk",
+    "command": "tuhi",
+      "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--talk-name=org.freedesktop.tuhi1",
+        "--own-name=org.freedesktop.tuhi1",
+        "--system-talk-name=org.bluez"
+      ],
+    "modules": [
+        {
+            "name": "pyxdg",
+            "buildsystem": "simple",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "git://anongit.freedesktop.org/xdg/pyxdg"
+                }
+            ],
+            "build-commands": [
+                "pip3 install --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} ."
+            ]
+        },
+        {
+            "name": "python-pyparsing",
+            "buildsystem": "simple",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/pyparsing/pyparsing/releases/download/pyparsing_2.4.2/pyparsing-2.4.2.tar.gz",
+                    "sha512": "27e5959eb1cf0c4d899746d2d32f5f000c3753278bdbbb670d24a077053e5c08caf8429f684186c502f6d9bf358702e0a8b3fea40cd2b50807cf02ea38c750dd"
+                }
+            ],
+            "build-commands": [
+                "pip3 install --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} ."
+            ]
+        },
+        {
+            "name": "python-svgwrite",
+            "buildsystem": "simple",
+            "sources": [
+                {
+                    "type": "git",
```

Comment:
We need a tag/commit here. Flathub requries reproducible builds

Reply:
all fixed, thanks.